Update "Lutris (Wine)" with more recent info (#169)
R2Northstar/NorthstarWiki: docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

## Lutris (Wine)

> 1. If you don't already have the game downloaded, You can use [this Lutris script](https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App), or [this one](https://lutris.net/games/titanfall-2/).

Reviewer: typo

```suggestion
1. If you don't already have the game downloaded, you can use [this Lutris script](https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App), or [this one](https://lutris.net/games/titanfall-2/).
```

> 2. Install the latest version of Northstar using [FlightCore](../installing-northstar/northstar-installers#geckoeidechse-flightcore), [Viper](../installing-northstar/northstar-installers#0negal-viper), or do it manually

Reviewer 2: personally think we should be recommending manual first & foremost over auto installers

Author: That part is copied from the Steam part of the guide. If you want I can change it there too though.

Reviewer: > personally think we should be recommending manual first & foremost over auto installers

Cringe take >:(

Reviewer 2: > > personally think we should be recommending manual first & foremost over auto installers
> 
> Cringe take >:(

![image](https://user-images.githubusercontent.com/116909003/236630466-c4f1c405-7585-4662-8bc3-2c528aeb7a2c.png)

   1. For manual install, download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
   2. Then extract all contents of the file to your Titanfall 2 folder ( Right click _Titanfall 2_ > Click _Browse files_ )
3. Add `-northstar` as a [launch option](../installing-northstar/troubleshooting.md#launch-opts) to Titanfall2

> 4. Disable in-game overlay: In EA App, click on the hamburger menu on the top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_ to OFF. In Origin, it's not necessary to turn off overlay, but doing so improves performance.

Reviewer 2: ```suggestion
4. Disable in-game overlay: In EA App, click on the hamburger menu on the top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_ to OFF.
```
we shouldn't be mentioning Origin anymore.

Author: I added the mentions to Origin because Gecko mentioned it earlier.

Reviewer: Yup, I'd keep it as we still have a bunch of people using it...

Reviewer 2: Origin is deprecated, and we should treat it as such and stop mentioning it, keeping it alive will just cause more issues in the long run.

Reviewer: I don't fully disagree but the reality is that right now (especially on Linux) we still have a bunch of people using it. Removing Origin from the wiki should be done in separate PR IMO

{% hint style="info" %}
Alternatively, you can open the EA App and launch _NorthstarLauncher.exe_ via _Run EXE inside Wine prefix_ and the in-game overlay will work, after several errors. However, the playtime won't be logged in the EA App.
{% endhint %}

> Now just launch the EA App/Origin through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.

Reviewer 2: ```suggestion
Now just launch the EA App through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.
```
no Origin nonsense

> Now just launch the EA App/Origin through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.

Reviewer 3: ```suggestion
Now just launch EA App/Origin through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.
```
We should probably get to one decided way to refer to the EA App. Earlier in the page, its referred to as just `EA App` without saying `The EA App`. The troubleshooting page is also guilty of this issue but it should be changed to be one way, starting here preferably while there's already an open pr for a page with the issue

> > **Note:** If using Origin, it might prompt you to log in and "set an installation folder for future downloads" on first launch. Just do those, close Origin, then launch the game again.

Reviewer 2: ```suggestion

```
_Origin begone_

## Lutris (Wine)

1. If you don't already have the game downloaded, install the game [from here.](https://lutris.net/games/titanfall-2/)
2. Install the latest version of Northstar using [FlightCore](../installing-northstar/northstar-installers#geckoeidechse-flightcore), [Viper](../installing-northstar/northstar-installers#0negal-viper), or do it manually

>    1. For manual install download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page

Reviewer 3: ```suggestion
   1. For manual install, download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
```

Reviewer 3: Minor grammar nitpick

   2. Then extract all contents of the file to your Titanfall 2 folder ( Right click _Titanfall 2_ > Click _Browse files_ )
3. Add `%command% -northstar` as a [launch option](../installing-northstar/troubleshooting.md#launch-opts) to Titanfall2

> 4. Disable in-game overlay: In EA App, click on hamburger menu at top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_

Reviewer 3: ```suggestion
4. Disable in-game overlay: In EA App, click on the hamburger menu on the top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_ to OFF
```

Reviewer 3: Minor grammar nitpicks, clarify on toggling overlay to off
